clash-lib/src/Clash/Core/Var.hs

```diff
@@ -8,7 +8,6 @@
 -}
 
 {-# LANGUAGE DeriveAnyClass        #-}
-{-# LANGUAGE DeriveGeneric         #-}
 {-# LANGUAGE ExplicitForAll        #-}
```

Comment:
ExplicitForAll is implied by ScopedTypeVariables which is now default (also implied by ExistentialQuantification, which is in turn implied by GADTs)